numpy/lib/tests/test_arraypad.py

```diff
@@ -82,6 +125,30 @@ def test_check_maximum_2(self):
             )
         assert_array_equal(a, b)
 
+    def test_check_maximum_stat_length(self):
```

Comment:
@WarrenWeckesser Holy smokes, you're right! I need to find a PEP8 checker that's smart enough to handle those cases, because the one I've been using flags them all.